Sources/ConsoleKit/Terminal/Terminal.swift

```diff
@@ -91,21 +91,23 @@ public final class Terminal: Console {
 
     /// See `Console`
     public func output(_ text: ConsoleText, newLine: Bool) {
-        var lines = 0
-        for fragment in text.fragments {
-            let strings = fragment.string.split(separator: "\n", omittingEmptySubsequences: false)
-            for string in strings {
-                let count = string.count
-                if count > size.width && count > 0 && size.width > 0 {
-                    lines += (count / size.width) + 1
-                }
-            }
-            /// add line for each fragment
-            lines += strings.count - 1
-        }
-        if newLine { lines += 1 }
-
-        didOutputLines(count: lines)
+		if !self.enableCommands {
```

Comment:
This is backwards; we want to do this only if commands are _enabled_, not disabled 😅. Not your fault; my wording when I suggested this was more than a little confusing, even to myself 😆.
```suggestion
		if self.enableCommands {
```